website/templates/project/contributors.mako

```diff
@@ -353,8 +356,10 @@
 
                 <!-- ko ifnot: contributor.canEdit() -->
                     <!-- ko if: canRemove -->
```

Comment:
[One](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/static/js/contribManager.js#L106-L108) is on `ContributorModel`, while the [other](https://github.com/CenterForOpenScience/osf.io/blob/ec285291ea829cf13ff47a8b3631474d35787baa/website/static/js/contribManager.js#L221-L223) is on `ContributorsViewModel `.